Sources/AccessibleAuthorLabel/AccessibleAuthorLabel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AccessibleAuthorLabel.swift
+//  AccessibleAuthorPlayground
+//
+//  Created by Christian Selig on 2021-12-21.
+//
+
+import UIKit
+
+/// A UILabel subclass that automatically generates better accessibility labels for strings that contain
+/// complex mathematical characters as a fancy way of styling text.
+public class AccessibleLabel: UILabel {
+    /// Statically (lazily) load our mapping plist file, so it's loaded only when first used and kept around afterwards.
+    static let characterMapping: [String: String] = {
```

Comment:
If we're keeping it in memory the entire time, where it's about 50 KB (and likely to grow some) it might be worth considering a different/more efficient data structure at that point since there's a lot of duplicated entries as is. (Or I'm just scared of RAM usage and micro-optimizing 😛)